Cargo.toml

```
@@ -40,7 +40,7 @@ minilp = [
 coin_cbc = { version = "0.1", optional = true, default-features = false }
 microlp = { version = "0.2.6", optional = true }
 lpsolve = { version = "0.1", optional = true }
-highs = { version = "1.7.0", optional = true }
+highs = { git = "https://github.com/rust-or/highs", optional = true }
```

I think we can just do
```suggestion
highs = { version = "1.8.0", optional = true }
```
now …